k8s/coordinator/development/cert-volume-mount.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: coordinator-deployment
+spec:
+  template:
+    spec:
+      volumes:
+        - name: tls-certificate
+          secret:
+            secretName: dev-coordinator
+            items:
+              - key: tls.key
+                path: tls.key
+                mode: 0400
+              - key: tls.crt
```

Comment:
```suggestion
              - key: tls.pem
```

Reply:
Actually the *key* is stored by our letsencrypt issuer as `tls.crt` inside the secrets, so we can't change the key to `tls.pem`, otherwise it'll reference a non-existing key. 

What can be done (and it's what I did) is to mount that `tls.crt` key to a `tls.pem` file. :)

Reply:
Ahh I see 👍